app/services/hyrax/statistics/works/by_resource_type.rb

```diff
@@ -5,7 +5,7 @@ class ByResourceType < Statistics::TermQuery
         private
 
           def index_key
-            Solrizer.solr_name("resource_type", :facetable)
+            Solrizer.solr_name("resource_type", :symbol)
```

Comment:
How about hard coding `resource_type_ssim` here?  Solrizer is deprecated and is going away, so this might just make it easier.